tests/Linter/ProcessLintingResultTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Linter;
+
+use PhpCsFixer\Linter\ProcessLintingResult;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Linter\ProcessLintingResult
+ */
+final class ProcessLintingResultTest extends TestCase
+{
+    public function testCheckOK()
+    {
+        $process = $this->prophesize('Symfony\Component\Process\Process');
+        $process
+            ->wait()
+            ->willReturn(0)
+        ;
+
+        $process
+            ->isSuccessful()
+            ->willReturn(true)
+        ;
+
+        $result = new ProcessLintingResult($process->reveal());
+        $result->check();
+        $result->check(); // call twice
+
+        $this->addToAssertionCount(1); // no exception, so we are good here
```

Comment:
maybe set expectation on how much times prophesized methods shall be called ? then, we will have assertions and no need for this trick

Reply:
-1, I don't want to test how often the (prophesized) methods are called internally.
PHPUnit really misses a default which I always went about like this. Maybe I reopen the discussion about this there some day.

Reply:
@SpacePossum , you don't need to count them, use `->shouldBeCalled()`.
if you don't call them, no need to prophesize them, right ?